### PR TITLE
ingested rocling 2019

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/xml/2019.rocling.xml [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+      <address>New Taipei City, Taiwan</address>
+      <month>October</month>
+      <year>2019</year>
+      <url hash="7cb428a0">2019.rocling-1</url>
+    </meta>
+    <frontmatter>
+      <url hash="f50c4de0">2019.rocling-1.0</url>
+    </frontmatter>
+    <paper id="1">
+      <title>基於特徵粒度之訓練策略於中文口語問答系統之應用(A Feature-granularity Training Strategy for <fixed-case>C</fixed-case>hinese Spoken Question Answering) [In <fixed-case>C</fixed-case>hinese]</title>
```
Comment (mjpost) [inline data/xml/2019.rocling.xml:19]: I suggest that we remove the `[In Chinese]` from the title, and just add a `<language>zhe</language>` tag instead.

Comment (mjpost) [reply data/xml/2019.rocling.xml:19]: (Do this for all papers)

**Comment by mjpost** [comment]
Also, this doesn't appear to be based on the latest `master`. Can you merge that in?

**Comment by mjpost** [comment]
(Nevermind, just did it, also fixed an error with ROCLING 2020)

File: data/xml/2019.rocling.xml (hunk 1 above)
Comment (xinru1414) [reply data/xml/2019.rocling.xml:19]: okay got it

**Review by mjpost — APPROVED** [review]
Looks good I think!

**Pull request merged**